components/ConvertSteps/StepperLayout.js

```diff
@@ -8,45 +8,41 @@ const smallCaps = css`
   font-size: 32px;
 `
 
-function getTitle(anjCount, antCount, stage, toAnj) {
-  const anjToAntTitle = (
+function getTitle(fromAmount, toAmount, stage, toAnj) {
+  const convertTitle = (
     <>
-      Convert {anjCount} <span css={smallCaps}>ANJ</span> to{' '}
-      <span css={smallCaps}>ANT</span>
+      Convert {fromAmount} <span css={smallCaps}>{toAnj ? 'ANT' : 'ANJ'}</span>{' '}
+      to <span css={smallCaps}>{toAnj ? 'ANJ' : 'ANT'}</span>
     </>
   )
-  const antToAnjTitle = (
-    <>
-      Convert {antCount} <span css={smallCaps}>ANT</span> to{' '}
-      <span css={smallCaps}>ANJ</span>
-    </>
-  )
-  const antToAnjSuccessTitle = (
-    <>
-      You successfully converted <br />
-      {antCount} <span css={smallCaps}>ANT</span> to {anjCount}{' '}
-      <span css={smallCaps}>ANJ</span>
-    </>
-  )
-  const anjToAntSuccessTitle = (
+
+  const successTitle = (
     <>
       You successfully converted <br />
-      {anjCount} <span css={smallCaps}>ANJ</span> to {antCount}{' '}
-      <span css={smallCaps}>ANT</span>
+      {fromAmount} <span css={smallCaps}>{toAnj ? 'ANT' : 'ANJ'}</span> to{' '}
+      {toAmount} <span css={smallCaps}>{toAnj ? 'ANJ' : 'ANT'}</span>
```

Comment:
This is might be more of a question for @dizzypaty, but what do you think about showing the token icons here?